gitea/docker-compose.yml

```diff
@@ -34,7 +34,7 @@ services:
         ipv4_address: $APP_GITEA_IP
 
   db:
-    image: mariadb:10.5.12@sha256:dfcba5641bdbfd7cbf5b07eeed707e6a3672f46823695a0d3aba2e49bbd9b1dd
+    image: mariadb:10.9.4
```

Comment:
Are you sure it's safe for the database to update from `10.5.12` to `10.9.4`?

Reply:
No. The mariadb image needs to be backtested.

Reply:
@pynixis Would you be able to kindly test this? I would suggest you install the current version of gitea and test this app update with the newer version of mariadb.